eclipse/src/saros/ui/util/CollaborationUtils.java

```diff
@@ -276,6 +237,32 @@ public void run() {
         });
   }
 
+  /**
+   * Returns the reference point objects representing the given container objects.
+   *
+   * @param referencePointContainers the container objects to represent as reference points
+   * @return the reference point objects representing the given container objects
+   */
+  private static Set<IReferencePoint> getReferencePoints(Set<IContainer> referencePointContainers) {
+    return referencePointContainers
+        .stream()
+        .map(EclipseReferencePointImpl::new)
+        .collect(Collectors.toSet());
+  }
+
+  /**
+   * Returns all projects that contain at least one of the given reference points.
+   *
+   * @param referencePoints the currently shared reference points
+   * @return all projects that contain at least one of the given reference points
+   */
+  private static Set<IProject> getReferencePointProjects(Set<IReferencePoint> referencePoints) {
```

Comment:
getProjects(FromReferencePoints) or convertToProjects ?

Reply:
Will rename to `getProjects(...)`.

Reply:
Done.